### DISCUSSION TITLE
Support for @counter-style

### DISCUSSION THREAD

**Discussion by ok-98** [vanilla-extract-css/vanilla-extract#1362] [Ideas]
A support for @counter-style ([https://developer.mozilla.org/en-US/docs/Web/CSS/@counter-style](https://developer.mozilla.org/en-US/docs/Web/CSS/@counter-style)) would be nice

**Comment by askoufis** [comment]
@kacper-olszanski Makes sense given it's now supported in all mainstream browser. Feel free to raise an issue so we can better track the feature request.
